pintracker/stateless/stateless.go

```diff
@@ -241,12 +258,16 @@ func (spt *Tracker) Track(ctx context.Context, c *api.Pin) error {
 			return nil // ongoing unpin
 		}
 		err := spt.unpin(op)
-		op.Cancel()
 		if err != nil {
 			op.SetError(err)
-		} else {
-			op.SetPhase(optracker.PhaseDone)
+			op.Cancel()
+			return nil
 		}
+
+		op.SetPhase(optracker.PhaseDone)
+		op.Cancel()
+		spt.optracker.Clean(ctx, op)
```

Comment:
Avoid refactorings, this is adding more lines.

Why is the operation being cleaned now?

Reply:
It is already in phase done. There is nothing left to do.
If all operations are removed right after they are done, the ticker (that cleans operations every 10 seconds) can be removed.